python/hail/dataset.py

```diff
@@ -2336,6 +2336,22 @@ def mendel_errors(self, output, fam):
         pargs = ['mendelerrors', '-o', output, '-f', fam]
         self.hc._run_command(self, pargs)
 
+    def min_rep(self):
+        """
+        Gives minimal, left-aligned representation of alleles. Note that this can change the variant position.
+
+        ** Examples **
+        1) Simple trimming of a multi-allelic site, no change in variant position
+        `1:10000:TAA:TAA,AA` => `1:10000:TA:T,A`
+
+        2) Trimming of a bi-allelic site leading to a change in position
+        `1:10000:AATAA,AAGAA` => `1:10002:T:G`
+
+        """
+        jvds = self._jvds.minrep()
```

Comment:
add in the try: / except: here, following the other methods in dataset.py.

The default py4j errors look horrible, so calling our wrapper method helps a lot.